adafruit_matrixportal/network.py

```diff
@@ -46,7 +46,7 @@
     )
     raise
 
-__version__ = "0.0.0-auto.0"
+__version__ = "1.9.4"
```

Comment:
Please leave the version at "0.0.0-auto.0" or the automatic versioning won't work.

Reply:
Will do.